src/telegraf.ts

```diff
@@ -77,6 +113,20 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
     throw err
   }
 
+  private handleTimeout = (ctx: C, middleware: Promise<void>): void => {
```

Comment:
How about just moving `handleTimeout` and `handleError` onto `DecayingQueue` to make `initializeQueue` unnecessary?

Reply:
That would've been good, I think